src/pages/iou/request/step/IOURequestStepParticipants.js

```diff
@@ -51,8 +55,22 @@ function IOURequestStepParticipants({
         IOUUtils.navigateToStartStepIfScanFileCannotBeRead(receiptFilename, receiptPath, () => {}, iouRequestType, iouType, transactionID, reportID);
     }, [receiptPath, receiptFilename, iouRequestType, iouType, transactionID, reportID]);
 
+    useEffect(() => {
+        if (!newIouType.current) {
+            return;
+        }
+        IOU.updateMoneyRequestTypeParams(routes, newIouType.current);
+        newIouType.current = null;
+    }, [routes, participants]);
+
     const addParticipant = useCallback(
-        (val) => {
+        (val, isSplit) => {
+            if (isSplit && iouType !== CONST.IOU.TYPE.SPLIT) {
+                newIouType.current = CONST.IOU.TYPE.SPLIT;
+            } else if (!isSplit && iouType === CONST.IOU.TYPE.SPLIT) {
+                newIouType.current = CONST.IOU.TYPE.REQUEST;
+            }
```

Comment:
I write that logic to consider the SEND case too. Currently, we still use the old component for sending money, but in case we migrate it, the case for SEND is already handled too.

`newIouType.current = isSplit ? CONST.IOU.TYPE.SPLIT : CONST.IOU.TYPE.REQUEST;`

If we do this, a SEND type will be changed to REQUEST.

So, I make sure that the new iou type can be changed to REQUEST only if the current type is SPLIT.
`else if (!isSplit && iouType === CONST.IOU.TYPE.SPLIT)`

Reply:
Added. Please check